pelias-geocoder.js

```diff
@@ -8,294 +13,429 @@ function PeliasGeocoder(opts) {
   this.opts.useFocusPoint = opts.useFocusPoint;
   this.opts.removeDuplicates = opts.removeDuplicates === undefined ? true : opts.removeDuplicates;
   this.opts.onSubmitOnly = opts.onSubmitOnly;
+
   if (opts.marker) {
     this.opts.marker = {};
     this.opts.marker.icon = opts.marker.icon || 'marker-15';
     this.opts.marker.anchor = opts.marker.anchor || 'bottom';
+    this.opts.marker.multiple = opts.marker.multiple !== undefined ? opts.marker.multiple : true;
+    this._customHtmlMarkers = [];
   }
+
   if (opts.wof) {
-    this.opts.wof = {}
+    this.opts.wof = {};
     this.opts.wof.url = opts.wof.url || 'https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/master/data/';
-    this.getWOFURL = opts.wof.getWOFURL || this.getDefaultWOFURLFunction();
     this.opts.wof.fillColor = opts.wof.fillColor || "rgba(200, 40, 32, 0.1)";
     this.opts.wof.fillOutlineColor = opts.wof.fillOutlineColor || "rgba(200, 40, 32, 0.7)";
+    this.getWOFURL = opts.wof.getWOFURL || this.getDefaultWOFURLFunction();
   }
+
   if (opts.params) {
     this.params = '';
-    for (var i in opts.params) {
-      this.params += '&' + i + '=' + opts.params[i];
+    for (var key in opts.params) {
+      if (opts.params.hasOwnProperty(key)) {
```

Comment:
Whye use hasOwnProperty in a for each keys ? This should be always true right ?

Reply:
It allows to only consider the properties attached to the object itself, and not all properties on the prototype chain (that can be added by someone else).

Reply:
:+1: